scripts/test.sh

```diff
@@ -2,6 +2,6 @@
 
 set -xe
 
-elixir --sname aeternity@localhost -S mix test
+MIX_ENV=test elixir --sname aeternity@localhost -S mix test
```

Comment:
This isn't really needed, since it's set by `mix test` automatically

Reply:
done, made implicit again